README.md

```diff
@@ -1,8 +1,7 @@
 # Getting Started with the .NET MAUI Linear Gauge Control
 
-A quick start .NET MAUI project that shows how to create and configure the Linear Gauge control of Syncfusion. This project also includes a code snippet to set a specific axis pointer value and customize the axis line height and width as well as how to add multiple axis ranges. 
+The [.NET MAUI Linear Gauge](https://www.syncfusion.com/maui-controls/maui-linear-gauge?utm_source=github&utm_medium=listing&utm_campaign=maui-Linear-gauge-github-samples) is a multipurpose data visualization control that displays numerical values on a linear scale either horizontally or vertically. It has a rich set of features, such as axis, ranges and pointers, that are fully customizable and extendable. A quick start .NET MAUI project that shows how to create and configure the Linear Gauge control of Syncfusion. This project also includes a code snippet to set a specific axis pointer value and customize the axis line height and width as well as how to add multiple axis ranges. 
```

Comment:
Remove unwanted content:
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-linear-gauge/assets/95272441/87568a63-8c18-45e8-9466-13f170df8eb4)

Reply:
Removed

Reply:
Removed